modulo_1/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GitSearch - Pesquisando Repositórios</title>
+    <meta name="description"
+          content="Página destinada à pesquisa de repositosrios contidos dentro do GitHub, busca atraves do nome do usuario">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="stylesheet" href="./assets/css/personalRepositories.css">
+    <link rel="stylesheet" href="./assets/css/personalInformation.css">
+    <link rel="stylesheet" href="./assets/css/searchBar.css">
+</head>
+
+<body>
+<section class="searchBar">
+    <div class="searchContainer">
+        <img src="./assets/img/gitsearch_2.png" alt="">
+        <input id="searchId" type="text" class="form " placeholder="Digite o usuário e precione enter..." required>
+    </div>
+</section>
+
+<section class="userInformation">

```

Comment:
**Dica:**
**Difference between 'section' and 'div' in 'aside' sidebar**
https://stackoverflow.com/questions/39232773/difference-between-section-and-div-in-aside-sidebar